integration-tests/.pytest.ini

```diff
@@ -0,0 +1,15 @@
+[pytest]
+addopts =
+    --capture=no
+    --code-highlight=yes
+    --color=yes
+    --strict-config
+    --strict-markers
+    --verbose
+env =
+    D:CLP_BUILD_DIR=../build
+    D:CLP_PACKAGE_DIR=../build/clp-package
```

Comment:
Since this integration test framework won't only be used for the package, it would be good to add a CLP_CORE_BINS_DIR environment variable for folks who only want to test the core binaries. The core binaries can be built with `task core` and the output will be written to `build/core`.